server/docker-entrypoint.sh

```diff
@@ -195,9 +195,8 @@ RUN_TYPE='STANDALONE'
 CONTAINER_SETTINGS="true"
 SERVER_OPTIONS=""
 SERVER_CONFIGURATION="clustered.xml"
-# Xms should always be set: https://developers.redhat.com/blog/2014/07/15/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-1/
-JAVA_OPTS="-Xms64m -Djava.net.preferIPv4Stack=true"
-PERCENT_OF_MEMORY_FOR_MX=70
+JAVA_OPTS="-Djava.net.preferIPv4Stack=true -XX:+DisableExplicitGC"
+PERCENT_OF_MEMORY_FOR_MX=50
```

Comment:
Above this setting our container was taken down by the OOM killer. The JVM overhead below 1 GB is tremendous.... See some more investigation on internal mailing lists